wmt/controllers/models.py

```diff
@@ -282,24 +308,34 @@ def GET(self, id, name):
         else:
             try:
                 component = models.get_model_component(int(id), name)
+                component['parameters']['_model_id'] = id
```

Comment:
Move this stuff into the `else` block. I think it's good practice to limit the amount of code in the `try` block.